cloudferrylib/os/actions/cinder_database_manipulation.py

```diff
@@ -62,3 +67,56 @@ def run(self, *args, **kwargs):
                 volume['instance_uuid'] = None
                 volume['attach_status'] = 'detached'
         self.get_resource().deploy(jsondate.dumps(data))
+
+
+class CopyVolumesData(action.Action):
```

Comment:
Please add docstring describing
 - Overview of the class
 - How to use it
 - Required config options
 - Dependent actions (what actions must follow)
 - Required actions (what actions this action depends on)